openupgrade_scripts/scripts/mrp/14.0.2.0/post-migration.py

```diff
@@ -89,6 +89,4 @@ def migrate(env, version):
     map_mrp_production_state_planned(env)
     generate_workorders_for_draft_orders(env)
     openupgrade.load_data(env.cr, "mrp", "14.0.2.0/noupdate_changes.xml")
-    openupgrade.delete_records_safely_by_xml_id(
-        env, ["mrp.mrp_routing_rule", "mrp.sequence_mrp_route"]
-    )
+    openupgrade.delete_records_safely_by_xml_id(env, ["mrp.mrp_routing_rule"])
```

Comment:
```suggestion
    openupgrade.delete_records_safely_by_xml_id(env, ["mrp.mrp_routing_rule"])
    if not openupgrade.is_module_installed(cr.cr, "mrp_routing"):
        openupgrade.delete_records_safely_by_xml_id(env, ["mrp.sequence_mrp_route"])
```

Reply:
Thanks @MiquelRForgeFlow, but `mrp_routing` is a brand new module in v. 14.0, so this check will always result true and delete the sequence.

Reply:
But when installing `mrp_routing`, the migration script of `mrp_routing` will not execute 🤔

Reply:
Mmm, good point, I'll correct that.

Reply:
Migration scripts for newly installed modules happen in the context of OpenUpgrade (if you load `openupgrade_framework`. Please put here the condition to find the module in the module list, which is likely that you are going to install the module.

Reply:
So `pre-migration.py` script it's called even in a newly installed module?

I don't know what the phrase `put here the condition to find the module in the module list` means, does it refer to the `openupgrade.is_module_installed` method?

Reply:
Yes and I mean:

`if env["ir.module.module"].search([("name", "=", "mrp_routing")]):`

Reply:
Wow, I'll try, thanks.